### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by huykaamaa** [tttapa/Control-Surface#949] [Show and tell]
This is my project
https://www.youtube.com/watch?v=7eGbP92QHzE
But sometime when setpoint change, fader suddenly go to max or min position, and can't control.
I check serial monior of nano it show: setpoint 1023 1022, and motor get hot.
When i press reset on nano everything ok
I try remove pro micro, only use hairless midi on nano but it still same problem.

**Comment by Trickster-git** [comment]
Great work!!!!!
Share your code.  :)
